src/parser.rs

```diff
@@ -54,10 +58,11 @@ fn parse_regex_string(input: &str) -> Res<&str, Node> {
 fn parse_terminal(input: &str) -> Res<&str, Node> {
     let (input, symbol) = preceded(
         complete::multispace0,
-        terminated(complete::alphanumeric1, complete::multispace0),
+        terminated(many1(alt((complete::alphanumeric1,tag("_")))), complete::multispace0),
```

Comment:
forget to `cargo fmt`?

Reply:
just formatted, didn't know about `cargo fmt` as I'm new to rust

Reply:
don't worry, it's just a good practice to maintain a huge library, nothing more than that :). Anyways, thanks for the format.